modules/python/src2/hdr_parser.py

```diff
@@ -595,6 +602,8 @@ def parse_func_decl(self, decl_str, mat="Mat", docstring=""):
 
                         if arg_type == "InputArray":
                             arg_type = mat
+                            if is_arithm_op_func and  arg_name in { "src1", "src2" }:
```

Comment:
I belive it is needed. Because these functions have not only `src1` `src2` but also `mask` as InputArray.
I think that the mask image should be removed from the correction target.

https://docs.opencv.org/4.x/d2/de8/group__core__array.html#ga10ac1bfb180e2cfda1701d06c24fdbd6

```
void cv::add	(
   InputArray 	src1,
   InputArray 	src2,
   OutputArray 	dst,
   InputArray 	mask = noArray(),
    int 	dtype = -1 
)		
Python:
cv.add(	src1, src2[, dst[, mask[, dtype]]]	) ->	dst
```

Reply:
It's true, but the solution looks very fragile at this point, because it relies on things that cannot be checked rather during runtime nor generation time...

Reply:
Thank you for your review, I don't think there's much of some problem without checking for arg_names.
Due to future implications, I will remove the verification code.